### PR TITLE
Create journal-of-computer-applications-in-archeology.csl

### PR DESCRIPTION
**Pull request by damnation333**
VIa https://forums.zotero.org/discussion/65737/style-request-journal-of-computer-applications-in-archaeology-caa
As pointed out in the thread, the style is very similar to citizen-science-theory-and-practice as they are both on Ubiquity press.
This journal is new and has not published anything. One to watch. I can imagine the missing dots are mistakes and we can make this style a dependent at some stage.

### CODE REVIEW

File: journal-of-computer-applications-in-archeology.csl [hunk 1]
```diff
@@ -0,0 +1,265 @@
...
+            </group>
+            <group delimiter=", ">
+              <text macro="issued" suffix=", "/>
+            </group>
+            <text macro="pages"/>
+          </group>
+        </else-if>
+        <else-if type="book graphic" match="any">
+          <group prefix=" " delimiter=". ">
+            <text variable="note"/>
```
Comment (damnation333) [inline journal-of-computer-applications-in-archeology.csl]: That was from the template. No clue. I've removed it.

File: journal-of-computer-applications-in-archeology.csl [hunk 2]
```diff
@@ -0,0 +1,265 @@
...
+      </substitute>
+    </names>
+  </macro>
+  <macro name="author-short">
+    <names variable="author" font-weight="normal">
+      <name form="short" and="text" delimiter-precedes-last="never"/>
+      <substitute>
+        <names variable="editor"/>
+        <names variable="translator"/>
+        <choose>
```
Comment (damnation333) [inline journal-of-computer-applications-in-archeology.csl]: thanks. all done.

File: journal-of-computer-applications-in-archaeology.csl [hunk 3]
```diff
@@ -0,0 +1,267 @@
...
+    <names variable="author">
+      <name and="text" delimiter-precedes-last="never" initialize-with=" " name-as-sort-order="all"/>
+      <label form="short" prefix=" "/>
+      <substitute>
+        <text macro="editor"/>
+        <choose>
+          <if type="report" match="any">
+            <text variable="publisher"/>
+          </if>
+        </choose>
```
Comment (rmzelle) [inline journal-of-computer-applications-in-archaeology.csl]: @damnation333, I assume it's better to add the `<names variable="translator"/>` here as well. It should always be possible to match the in-text citation against the bibliographic entry, so the author-macros almost always need the exact same substitution logic.

**Comment by rmzelle** [comment]
Thanks!

**Pull request merged**